changelog/17199.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+secrets/gcp: Update `github.com/hashicorp/vault-plugin-secrets-gcpkms` to `v0.13.0`.
```

Comment:
Oh shoot, I meant to suggest a CL update for what actually changed in the plugin itself instead of this version bump in Vault. The change in the plugin itself is that we updated its google.golang.org/api dependency to v0.83.0. I think the CL entry that I placed in my other comment is the one we need.

Exactly like this:

\```release-note:improvement
secrets/gcpkms: Updates \`google.golang.org/api\` dependency  to \`v0.83.0\`.
\```

Reply:
Fixed in #17202